jbpm-flow/src/main/java/org/jbpm/process/core/timer/impl/quartz/QuertzUtils.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.process.core.timer.impl.quartz;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.kie.internal.runtime.manager.RuntimeManagerRegistry;
+import org.quartz.impl.jdbcjobstore.StdJDBCConstants;
+
+public class QuertzUtils implements StdJDBCConstants {
```

Comment:
Just 2 things - I was searching for QuartzUtils but then I realized there is a typo in Qu**e**rtzUtils. The second thing is that since this is a util class, can we make methods and constants static? Then we don't have to initialize it inside Quartz delegates.

Reply:
the point here is that it implements the StdJDBCConstants as the extensions refer to other fields for the interface. I made it a class to keep the door open as it might need more work down the road.
Will fix the type on class name